websites/B/Brainly/presence.ts

```diff
@@ -0,0 +1,128 @@
+      startTimestamp: Date.now()
+    };
+    presence.setActivity(presenceData);
+  } else if (document.location.pathname.includes("/subject/history")) {
+    const presenceData: PresenceData = {
+      details: "Viewing questions",
+      largeImageKey: "logo",
+      state: "History",
+      smallImageKey: "qmark",
+      startTimestamp: Date.now()
+    };
+    presence.setActivity(presenceData);
+  } else if (document.location.pathname.includes("/subject/english")) {
+    const presenceData: PresenceData = {
+      details: "Viewing questions",
+      largeImageKey: "logo",
+      state: "English",
+      smallImageKey: "qmark",
+      startTimestamp: Date.now()
+    };
+    presence.setActivity(presenceData);
+  } else if (document.location.pathname.includes("/subject/geography")) {
+    const presenceData: PresenceData = {
+      details: "Viewing questions",
+      largeImageKey: "logo",
+      state: "Geography",
+      smallImageKey: "qmark",
+      startTimestamp: Date.now()
+    };
+    presence.setActivity(presenceData);
+  } else if (document.location.pathname.includes("/subject/biology")) {
+    const presenceData: PresenceData = {
+      details: "Viewing questions",
+      largeImageKey: "logo",
+      state: "Biology",
+      smallImageKey: "qmark",
+      startTimestamp: Date.now()
+    };
+    presence.setActivity(presenceData);
+  } else if (document.location.pathname.includes("/subject/physics")) {
+    const presenceData: PresenceData = {
+      details: "Viewing questions",
+      largeImageKey: "logo",
+      state: "Physics",
+      smallImageKey: "qmark",
+      startTimestamp: Date.now()
+    };
+    presence.setActivity(presenceData);
+  } else if (document.location.pathname.includes("/subject/chemistry")) {
+    const presenceData: PresenceData = {
+      details: "Viewing questions",
+      largeImageKey: "logo",
+      state: "Chemistry",
+      smallImageKey: "qmark",
+      startTimestamp: Date.now()
+    };
+    presence.setActivity(presenceData);
+  } else if (document.location.pathname.includes("/subject/social_sciences")) {
+    const presenceData: PresenceData = {
+      details: "Viewing questions",
+      largeImageKey: "logo",
+      state: "Social Science",
+      smallImageKey: "qmark",
+      startTimestamp: Date.now()
+    };
+    presence.setActivity(presenceData);
+  } else if (document.location.pathname.includes("/subject/environmental_sciences")) {
+    const presenceData: PresenceData = {
+      details: "Viewing questions",
+      largeImageKey: "logo",
+      state: "Environmental Science",
+      smallImageKey: "qmark",
+      startTimestamp: Date.now()
+    };
+    presence.setActivity(presenceData);
+  } else if (document.location.pathname.includes("/all-questions")) {
+    const presenceData: PresenceData = {
+      details: "Viewing all questions",
+      largeImageKey: "logo",
+      smallImageKey: "qmark",
+      startTimestamp: Date.now()
+    };
+    presence.setActivity(presenceData);
```

Comment:
big protip, use one presenceData variable then just add things by e.g.
```javascript
presenceData.startTimestamp = Date.now();
presenceData.smallImageKey = "qmark";
``` 
instead of creating object every time :/